pkg/bootstrapper/bootstrapper.go

```diff
@@ -178,13 +183,24 @@ func NewWinNodeBootstrapper(k8sInstallDir, ignitionFile, kubeletPath string, cni
 		}
 	}
 
-	// If there is already a kubelet service running, find it
-	if ksvc, err := svcMgr.OpenService(KubeletServiceName); err == nil {
-		bootstrapper.kubeletSVC = ksvc
+	bootstrapper.service, err = newService(svcMgr)
+	if err != nil {
+		return nil, err
 	}
+
+	// If there is already a kubelet service running, find it and assign it
+	bootstrapper.service.setService(KubeletServiceName)
```

Comment:
Why are we not capturing the error here ?

Reply:
the reason is (// If there is already a kubelet service running, find it and assign it) if there isn't a kubelet service already present we don't need to make sure it is assigned here. It will get created in createService and be assigned there.

Reply:
I think we should be careful about doing this, what if there is a kubelet service running and we cannot open it for whatever reason? (permissions or something).
We would need to capture that error. 

If setService is being used in this way I think it should either be fed a service object, or we should check that the service exists before calling it, then only call it if the service does exist, and error out if it returns an error

Reply:
I am capturing the error for setService(), it is not possible to do it in this particular call as the service is initialized here and service object will always be nil, hence we cannot validate it here.